bin/copy-assets.js

```diff
@@ -10,27 +13,42 @@ function start() {
   console.log("start: copy assets");
   const projectPath = path.resolve(__dirname, "..");
   const mcModulePath = "devtools/client/shared/components/reps";
-  const mcPath = "./firefox";
 
-  symlinkTests({ projectPath, mcModulePath });
+  const envConfig = getConfig();
+  setConfig(envConfig);
 
+  const mcPath = getValue("firefox.mcPath");
+
+  console.log("  output path is:", mcPath);
+  console.log("  copying load-reps...");
   copyFile(
     path.resolve(projectPath, "src/reps/load-reps.js"),
     path.join(mcPath, mcModulePath, "load-reps.js"),
     {cwd: projectPath}
   );
 
+  console.log("  copying tests...");
+  copyFile(
+    path.resolve(projectPath, "src/test/mochitest/"),
+    path.join(mcPath, mcModulePath, "test/mochitest/"),
+    {cwd: projectPath}
+  );
```

Comment:
perhaps use a similar condition so that it's possible to symlink

https://github.com/arthur801031/debugger.html/blob/white_space/bin/copy-assets.js#L54-L61

The rationale is that the symlinking workflow is much faster.